packages/tupaia-web/src/features/PDFExportDashboardItem/PDFExportDashboardItem.tsx

```diff
@@ -154,39 +149,36 @@ export const PDFExportDashboardItem = ({
       $isPreview={isPreview}
       $previewZoom={previewZoom}
     >
-      <PDFExportHeader>{entityName}</PDFExportHeader>
-      <A4PageContent>
-        <DashboardTitleContainer>
-          <DashboardNameText>{activeDashboard?.name}</DashboardNameText>
-          <Divider />
-        </DashboardTitleContainer>
-        <Wrapper>
-          <Title>{title}</Title>
-          {reference && <ReferenceTooltip reference={reference} />}
-          {period && <ExportPeriod>{period}</ExportPeriod>}
-          <ExportContent $hasData={data && data?.length > 0}>
-            <DashboardItemContext.Provider
-              value={{
-                config: {
-                  ...config,
-                  presentationOptions: {
-                    ...config.presentationOptions,
-                    exportWithTable: true,
-                  },
+      <PDFExportHeader imageUrl={projectLogoUrl} imageDescription={projectLogoDescription}>
```

Comment:
This line is the only actual change in this sea of red/green. I just removed the `<A4PageContent>` wrapper so all this is one tab shallower now